packages/api-client/src/granules.ts

```diff
@@ -330,6 +330,38 @@ export const listGranules = async (params: {
   });
 };
 
+/**
+ * Bulk operations on granules stored in cumulus
+ * POST /granules/bulk
+ * @param {Object} params             - params
+ * @param {Object} params.body       - body to pass the API lambda
+ * @param {Function} params.callback  - async function to invoke the api lambda
+ *                                      that takes a prefix / user payload.  Defaults
+ *                                      to cumulusApiClient.invokeApifunction to invoke the
+ *                                      api lambda
+ * @returns {Promise<Object>}         - the response from the callback
+ */
+export const bulkGranules = async (params: {
+  prefix: string,
+  body: unknown,
+  callback?: InvokeApiFunction
+}): Promise<ApiGatewayLambdaHttpProxyResponse> => {
+  const { prefix, body, callback = invokeApi } = params;
+
+  return callback({
+    prefix: prefix,
+    payload: {
+      httpMethod: 'POST',
+      resource: '/{proxy+}',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      path: '/granules/bulk',
+      body: JSON.stringify(body),
```

Comment:
Would `object` be a more accurate type for `body`? I guess `stringify` doesn't care and will just make anything a string so it probably doesn't matter.